### DISCUSSION TITLE
Dynamic picklist field value on advanced block module - inventory line ?

### DISCUSSION THREAD

**Discussion by zimart** [YetiForceCompany/YetiForceCRM#17323] [How to]
Hello, is it possible to obtain dynamic picklist fields value  - according to the conditions set in the main module (PickListDependency) ?

conditions...

![obraz](https://github.com/YetiForceCompany/YetiForceCRM/assets/11014419/d97838bd-a994-401e-89c7-9ebbda8f0f43)

--

![yeti_dynamic_picklist](https://github.com/YetiForceCompany/YetiForceCRM/assets/11014419/cc35503b-1703-4b0e-a68f-cd04aa923f96)

Maybe someone will show me a way?

**Comment by mariuszkrzaczkowski** [comment]
No
